tests/graphical.rs

```diff
@@ -803,18 +815,46 @@ fn related() -> Result<(), MietteError> {
  3 │     here
    ╰────
   help: try doing it better next time?
-
-Error: oops::my::bad
-
-  × oops!
-   ╭─[bad_file.rs:1:1]
- 1 │ source
-   · ───┬──
-   ·    ╰── this bit here
- 2 │   text
-   ╰────
-  help: try doing it better next time?
-
+╭─There were 2 related diagnostics:
+├─ 1.Error: oops::my::bad
+│ 
+│ 
+│   × oops!
+│    ╭─[bad_file.rs:1:1]
+│  1 │ source
+│    · ───┬──
+│    ·    ╰── this bit here
+│  2 │   text
+│    ╰────
+│   help: try doing it better next time?
+│ ╭─There were 1 related diagnostics:
+│ ├─ 1.Error: oops::my::bad
+│ │
+│ │
+│ │   × oops!
+│ │    ╭─[bad_file.rs:1:1]
+│ │  1 │ source
+│ │    · ───┬──
+│ │    ·    ╰── this bit here
+│ │  2 │   text
+│ │    ╰────
+│ │   help: try doing it better next time?
+│ │ ├─There were 0 related diagnostics:
+│ │
+│ 
+├─ 2.Error: oops::my::bad
+│ 
+│ 
+│   × oops!
+│    ╭─[bad_file.rs:1:1]
+│  1 │ source
+│    · ───┬──
+│    ·    ╰── this bit here
+│  2 │   text
+│    ╰────
+│   help: try doing it better next time?
+│ ├─There were 0 related diagnostics:
```

Comment:
I'd expect this line to simply be omitted when there are no related diagnostics (what I expect to be the most common case to be fair).